inst/extdata/emissions/EDGAR/EDGAR_nation.csv

```diff
@@ -0,0 +1 @@
+# File: EDGAR_nation.csv,
# Title: Mapping from iso to ISO_A3,
# Units: NA,
iso,ISO_A3
afg,AFG
alb,ALB
dza,DZA
and,AND
ago,AGO
aia,AIA
aus,AUS
atg,ATG
arg,ARG
arm,ARM
abw,ABW
aus,AUS
aut,AUT
aze,AZE
bhs,BHS
bhr,BHR
bgd,BGD
brb,BRB
blr,BLR
bel,BEL
blz,BLZ
ben,BEN
bmu,BMU
btn,BTN
bih,BIH
bwa,BWA
bra,BRA
vgb,VGB
brn,BRN
bgr,BGR
bfa,BFA
bdi,BDI
khm,KHM
can,CAN
cpv,CPV
cym,CYM
caf,CAF
tcd,TCD
chl,CHL
chn,CHN
cxr,CXR
col,COL
com,COM
cog,COG
cok,COK
cri,CRI
civ,CIV
hrv,HRV
cub,CUB
cyp,CYP
cze,CZE
cze,CZE
prk,PRK
cod,COD
vnm,VNM
dnk,DNK
dji,DJI
dma,DMA
dom,DOM
pak,PAK
ecu,ECU
egy,EGY
slv,SLV
gnq,GNQ
eri,ERI
est,EST
eth,ETH
eth,ETH
fro,FRO
flk,FLK
deu,DEU
fsm,FSM
mys,MYS
fji,FJI
fin,FIN
yem,YEM
deu,DEU
pan,PAN
yem,YEM
fra,FRA
cog,COG
guf,GUF
vnm,VNM
pyf,PYF
sen,SEN
gab,GAB
gmb,GMB
geo,GEO
deu,DEU
gha,GHA
gib,GIB
grc,GRC
grl,GRL
grd,GRD
glp,GLP
gtm,GTM
gin,GIN
gnb,GNB
guy,GUY
hti,HTI
hnd,HND
hkg,HKG
hun,HUN
isl,ISL
ind,IND
idn,IDN
irq,IRQ
irl,IRL
irn,IRN
isr,ISR
ita,ITA
jam,JAM
jpn,JPN
jpn,JPN
jor,JOR
kaz,KAZ
ken,KEN
kir,KIR
kwt,KWT
kgz,KGZ
lao,LAO
lva,LVA
lbn,LBN
vgb,VGB
lbr,LBR
lby,LBY
ltu,LTU
lux,LUX
mac,MAC
mkd,MKD
mdg,MDG
mwi,MWI
mys,MYS
mdv,MDV
mli,MLI
mlt,MLT
mhl,MHL
mtq,MTQ
mrt,MRT
mus,MUS
mex,MEX
mng,MNG
msr,MSR
mar,MAR
scg,SCG
moz,MOZ
mmr,MMR
nam,NAM
nru,NRU
npl,NPL
ant,ANT
ant,ANT
nld,NLD
ncl,NCL
nzl,NZL
nic,NIC
ner,NER
nga,NGA
niu,NIU
nor,NOR
pse,PSE
omn,OMN
plw,PLW
pak,PAK
plw,PLW
pan,PAN
pan,PAN
png,PNG
pry,PRY
mys,MYS
per,PER
phl,PHL
bol,BOL
pol,POL
prt,PRT
pri,PRI
qat,QAT
cmr,CMR
kor,KOR
mda,MDA
vnm,VNM
reu,REU
zwe,ZWE
rom,ROM
rus,RUS
rwa,RWA
rwa,RWA
jpn,JPN
mys,MYS
shn,SHN
lca,LCA
wsm,WSM
stp,STP
mys,MYS
sau,SAU
sen,SEN
scg,SCG
syc,SYC
sle,SLE
sgp,SGP
svk,SVK
svn,SVN
slb,SLB
som,SOM
zaf,ZAF
esp,ESP
lka,LKA
kna,KNA
kna,KNA
spm,SPM
vct,VCT
sdn,SDN
sur,SUR
swz,SWZ
swe,SWE
che,CHE
syr,SYR
twn,TWN
tjk,TJK
ken,KEN
tha,THA
tls,TLS
tgo,TGO
ton,TON
tto,TTO
tun,TUN
tur,TUR
tkm,TKM
tca,TCA
uga,UGA
ukr,UKR
are,ARE
gbr,GBR
kor,KOR
tza,TZA
usa,USA
ury,URY
arm,ARM
aze,AZE
blr,BLR
est,EST
geo,GEO
kaz,KAZ
kgz,KGZ
ltu,LTU
lva,LVA
mda,MDA
rus,RUS
tjk,TJK
tkm,TKM
ukr,UKR
uzb,UZB
uzb,UZB
vut,VUT
ven,VEN
vnm,VNM
wlf,WLF
esh,ESH
yem,YEM
bih,BIH
hrv,HRV
mkd,MKD
scg,SCG
svn,SVN
scg,SCG
zmb,ZMB
tza,TZA
zwe,ZWE
umi,UMI
vir,VIR
bvt,BVT
sgs,SGS
myt,MYT
lso,LSO
rom,ROU
iot,IOT
asm,ASM
atf,ATF
cck,CCK
gum,GUM
hmd,HMD
mnp,MNP
nfk,NFK
pcn,PCN
tkl,TKL
tuv,TUV
```

Comment:
Take a look at the wiki, Kate replaced EDGAR nation with 3 lines of code so we don't need this file.